src/overlays/actors/ovl_Arrow_Ice/z_arrow_ice_gfx.c

```diff
@@ -0,0 +1,190 @@
+#include "z_arrow_ice.h"
```

Comment:
This file needs to be extracted from the ROM using ZAPD, as we're discussing: we do not want this data in the repo (and OoT will extract it too).